src/pages/GenericErrorPage/index.js

```diff
@@ -0,0 +1,91 @@
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
+import compose from '../../libs/compose';
+import * as Session from '../../libs/actions/Session';
+import variables from '../../styles/variables';
+import styles from '../../styles/styles';
+import ErrorBodyText from './ErrorBodyText';
+import TextLink from '../../components/TextLink';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+
+    /** Function call on Refresh button click */
+    onRefresh: PropTypes.func.isRequired,
+};
+
+const genericErrorPage = (props) => {
+    const onSignOutClick = () => {
+        Session.signOut();
+        props.onRefresh();
+    };
+
+    return (
+        <View style={[styles.flex1, styles.pv10, styles.ph5, styles.errorPageContainer]}>
+            <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                <View style={styles.alignItemsStart}>
+                    <View style={styles.mb5}>
+                        <Icon
+                            src={Expensicons.Bug}
+                            height={variables.componentSizeNormal}
+                            width={variables.componentSizeNormal}
+                            fill={defaultTheme.iconSuccessFill}
+                        />
+                    </View>
+                    <View style={styles.mb5}>
+                        <Text style={[styles.headerText, styles.textXXLarge]}>
+                            {props.translate('genericErrorPage.title')}
+                        </Text>
+                    </View>
+                    <View style={styles.mb5}>
+                        <ErrorBodyText />
+                        <Text>
+                            {props.translate('genericErrorPage.body.helpTextConcierge')}
+                            <TextLink href={`mailto:${CONST.EMAIL.CONCIERGE}`} style={[styles.link]}>
+                                {CONST.EMAIL.CONCIERGE}
+                            </TextLink>
+                        </Text>
+                    </View>
+                    <View style={[styles.dFlex, styles.flexRow, styles.w100]}>
+                        <View style={[styles.flex1, styles.flexRow, styles.justifyContentBetween]}>
+                            <Button
+                                success
+                                small={props.isSmallScreenWidth}
+                                onPress={props.onRefresh}
+                                text={props.translate('genericErrorPage.refresh')}
+                            />
+                            <Button
+                                small={props.isSmallScreenWidth}
+                                onPress={onSignOutClick}
+                                text={props.translate('initialSettingsPage.signOut')}
```

Comment:
If both buttons are left-aligned then they will look better. You can use margin to add gap between them.

Reply:
Added.